RocketChat/Rocket.Chat
# [FIX] "Read Only" and "Allow Reacting" system messages are missing in rooms

## Proposed changes (including videos or screenshots)


 - Add system message to notify changes on the **"Read Only"** setting;
 - Add system message to notify changes on the **"Allow Reacting"** setting;
 - Fix "Allow Reacting" setting's description (updated from "Only authorized users can write new messages" to "Only authorized users can react to messages").
![system-messages](https://user-images.githubusercontent.com/36537004/130883527-9eb47fcd-c8e5-41fb-af34-5d99bd0a6780.PNG)




## Issue(s)

[Task - ClickUp](https://app.clickup.com/t/5k5qvt)

## Steps to test or reproduce

1. Create a new channel (not read-only);
2. Go to **Room Information > Edit > Read Only** and turn this setting on. **_Expected Behavior:_** a system message with the text "Room set as Read Only by _your.user_" should be sent;
3. Turn the "Read Only" setting off. **_Expected Behavior:_** a system message with the text "Room added writing permission by _your.user_" should be sent;
4. Turn the "Read Only" setting on again and then turn the "Allow Reacting" setting on. **_Expected Behavior:_** a system message with the text "Room allowed reacting by matheus.barbosa by _your.user_" should be sent;
5. Turn the "Allow Reacting" setting off. **_Expected Behavior:_** a system message with the text "Room disallowed reacting by _your.user_" should be sent;
**IMPORTANT:** Save the changes after changing each setting (the system messages will only be sent _after_ saving).

## Further comments
